wurst/systems/items/spellbooks.wurst

```diff
@@ -11,6 +11,7 @@ import InventoryEvent
 import ItemExtensions
 import UnitExtensions
 
+// This is a fix for the Horn of Mammoth ability bug, sometime when you click on the horn, there is no ability available
```

Comment:
Don't refer to Mammoth of Horn directly. Can you just describe the general bug for spellbook items?

Reply:
"This is a fix for opening a spellbook on item click, sometime the spellbook would open but didn't show its abilities, in this case, the unit needed to drop the item and pick it up again"

Is this ok?